docs/api-docs/partner/subscriptions/foundation-guide.md

```diff
@@ -71,60 +71,67 @@ To develop and test the app, you'll install it on your [sandbox store](#configur
 
 Further steps in this guide require access to the Dev Portal, so keep it handy.
 
-## Stripe setup
+## Configure Stripe
 
-BigCommerce has configured the app to connect multiple BC Stores into one instance. Because of this, you’ll need two Stripe accounts. One for the Connect account which the app will use, and another, which is what the merchant connects to the app and BC store (in the payments area) themselves.
+We've designed this integration to use Stripe Connect so that your app can take payments with separate Stripe accounts for each of your merchants' stores. Because of this, you'll need two Stripe accounts. One for the app's Connect account, and another, which is what the merchant connects to the app and BC store (in the payments area) themselves.
```

Comment:
```suggestion
We've designed this integration to use Stripe Connect so that your app can take payments with separate Stripe accounts for each of your merchants' stores. Because of this, you'll need two Stripe accounts. One is for the app's Connect account, and another is what the merchant connects to the app and BC store (in the payments area) themselves.
```